foxpuppet/windows/browser/window.py

```diff
@@ -32,6 +32,14 @@ def notification(self):
         except NoSuchElementException:
             return None  # no notification is displayed
 
+    @property
+    def tabs(self):
```

Comment:
To the user, they don't need to access the tabbar directly, so I figured don't let them even know its there.